### ISSUE TITLE
[dev.icinga.com #9874] Add option to windows installer to disable inclusion of conf.d directory

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga2#3246]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/9874

**Created by twidhalm on 2015-08-11 09:11:02 +00:00**

Assignee: _(none)_
Status: _New_ 
Target Version: _Backlog_
Last Update: _2016-04-01 08:44:49 +00:00 (in Redmine)_

    Backport?: Not yet backported
    Include in Changelog: 1

---

Hi,

When deploying a Windows agent, that will get his configuration synced from the config master you have to remove the inclusion of `conf.d` to avoid duplicate items. It would be nice to have an option in the installer to avoid using the default configuration and only rely on configuration created during the setup and synced from the master.

Basically an option in the graphical installer to remove / comment the `include conf.d` line would be sufficient.

Cheers,
Thomas

**Comment by Crunsher** [comment]
This makes sense as long as Windows can't be a master instance, maybe that's something for @N-o-X

**Comment by mcktr** [comment]
@Crunsher I guess this depends on my PR #5426, as @dnsmichi mentioned adding a config parameter to the node setup makes it possible to add a checkbox to the windows installer.

**Comment by dnsmichi** [comment]
Yep, that was an imaginary dependency by this Windows issue. The global zone thing is resolved with the addition of default configs to the packages.
